src/components/MainFooter.vue

```diff
@@ -1,5 +1,17 @@
 <template>
-    <footer>
-        <h1>This is a footer</h1>
+  <div id="bot">
+    <footer>  
+
+            <h1>This is a footer,and now its gainning in lnength and losing typng strenth</h1> 
+
     </footer>
-</template>
\ No newline at end of file
+  </div>
+</template>
+
+<style>
```

Comment:
Might need scoped to only be applied to this element i.e
`<style scoped>`

Reply:
I suppose non-scoped styling could be used in conjunction with vue files to include certain stylings based on context. Probably not what we're looking for right now however.